### PR TITLE
feat(deployment): Add basic-auth to keycloak

### PR DESCRIPTION
**Pull request by theosanderson**
Adds basic auth for keycloak for use on staging with a sealed secret

https://basic-auth-keycloak.loculus.org/

loculus / widetailpotato

### CODE REVIEW

File: kubernetes/loculus/values_preview_server.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 createTestAccounts: true
 secrets:
+  basicauth:
```
Comment (theosanderson) [inline kubernetes/loculus/values_preview_server.yaml:3]: revert before merging

**Comment by corneliusroemer** [comment]
Ingest and/or prepro seem to fail? Don't see any sequences. We might have to add basic auth support to the pipeline's keycloak key requests?

**Comment by corneliusroemer** [comment]
Ah, were using public DNS instead of kubernetes internal one for keycloak url. Switching out the URL to use kubernetes internal one should be enough to fix prepro and ingest.

**Comment by corneliusroemer** [comment]
@theosanderson Anya and I fixed the issue by using k8s-internal DNS now, tested and works. I've also reverted the basicauth, so will merge this and make PP PR so we can protect staging soonish. Anya and I will coordinate this

**Pull request merged**